### ISSUE TITLE
About Avalanche C Chain

### ISSUE THREAD

**Issue by Wangmmx** [ava-labs/avalanchego#824]
Hello, I am doing Cchain support of exchange, and I have some questions: 

1. I have supported Xchain before,  it is utxo mode, I see Cchain is of eth type, these two chains are completely different?
2. I have successfully initiated a Cchain transaction, and the broadcast reported that the balance is insufficient. Can the balance on the Xchain be transferred to Cchain? If not, how to obtain test coins, etc. 
3. I would like to ask if the verso project will be supported on Cchain, is it a token of Cchain? So if I want to support verso,I must first support Cchain, right? 

Looking forward to your reply, thanks

**Comment by danlaine** [comment]
> I have supported Xchain before, it is utxo mode, I see Cchain is of eth type, these two chains are completely different?

Yes, the X-Chain uses a UTXO model, and the C-Chain, which uses the Ethereum Virtual Machine, uses an account model.

> I have successfully initiated a Cchain transaction, and the broadcast reported that the balance is insufficient. Can the balance on the Xchain be transferred to Cchain? If not, how to obtain test coins, etc.

X-Chain funds can be transferred to the C-Chain. See [here](https://docs.avax.network/build/tutorials/platform/transfer-avax-between-x-chain-and-c-chain). To get AVAX on the Avalanche testnet (which is named Fuji), use the [faucet](https://faucet.avax-test.network/).

> I would like to ask if the verso project will be supported on Cchain, is it a token of Cchain? So if I want to support verso,I must first support Cchain, right?

I'm not familiar with Verso, but if it is an EVM smart contract based token then it can be launched on Avalanche. Ethereum smart contracts work on Avalanche with minimal or no changes required.

If you are working with an exchange, and you'd like additional technical support, please email me at `daniel.laine@avalabs.org`.
